### ISSUE TITLE
Label-studio crashes on start 1.7.2 on macOS

### ISSUE THREAD

**Issue by joduss** [heartexlabs/label-studio#3955]
**Describe the bug**
When starting label-studio 1.7.2 macOS, installed via pip in a conda environment, a crash occurs.
This is not happening with label-studio 1.7.1

**To Reproduce**
- Create a new conda environment (any supported python version) and activate it
- pip install label-studio
- run 'label-studio' in the terminal

**Expected behavior**
No crash should occur

**Environment (please complete the following information):**
 - OS: macOS 13.2.1
 - Label Studio Version 1.7.2

**Additional context**

<details>
<summary>Stacktrace</summary>

```
Traceback (most recent call last):
  File "/usr/local/anaconda3/envs/labelstudio/bin/label-studio", line 8, in <module>
    sys.exit(main())
  File "/usr/local/anaconda3/envs/labelstudio/lib/python3.8/site-packages/label_studio/server.py", line 296, in main
    _setup_env()
  File "/usr/local/anaconda3/envs/labelstudio/lib/python3.8/site-packages/label_studio/server.py", line 40, in _setup_env
    application = get_wsgi_application()
  File "/usr/local/anaconda3/envs/labelstudio/lib/python3.8/site-packages/django/core/wsgi.py", line 12, in get_wsgi_application
    django.setup(set_prefix=False)
  File "/usr/local/anaconda3/envs/labelstudio/lib/python3.8/site-packages/django/__init__.py", line 19, in setup
    configure_logging(settings.LOGGING_CONFIG, settings.LOGGING)
  File "/usr/local/anaconda3/envs/labelstudio/lib/python3.8/site-packages/django/conf/__init__.py", line 82, in __getattr__
Starting new HTTPS connection (1): o227124.ingest.sentry.io:443
    self._setup(name)
  File "/usr/local/anaconda3/envs/labelstudio/lib/python3.8/site-packages/django/conf/__init__.py", line 69, in _setup
    self._wrapped = Settings(settings_module)
  File "/usr/local/anaconda3/envs/labelstudio/lib/python3.8/site-packages/django/conf/__init__.py", line 170, in __init__
    mod = importlib.import_module(self.SETTINGS_MODULE)
  File "/usr/local/anaconda3/envs/labelstudio/lib/python3.8/importlib/__init__.py", line 127, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 1014, in _gcd_import
  File "<frozen importlib._bootstrap>", line 991, in _find_and_load
  File "<frozen importlib._bootstrap>", line 975, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 671, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/usr/local/anaconda3/envs/labelstudio/lib/python3.8/site-packages/label_studio/core/settings/label_studio.py", line 48, in <module>
    from label_studio.core.utils.common import collect_versions
  File "/usr/local/anaconda3/envs/labelstudio/lib/python3.8/site-packages/label_studio/core/utils/common.py", line 41, in <module>
    from django.contrib.postgres.operations import TrigramExtension, BtreeGinExtension
  File "/usr/local/anaconda3/envs/labelstudio/lib/python3.8/site-packages/django/contrib/postgres/operations.py", line 1, in <module>
    from django.contrib.postgres.signals import (
  File "/usr/local/anaconda3/envs/labelstudio/lib/python3.8/site-packages/django/contrib/postgres/signals.py", line 3, in <module>
    import psycopg2
  File "/usr/local/anaconda3/envs/labelstudio/lib/python3.8/site-packages/psycopg2/__init__.py", line 51, in <module>
    from psycopg2._psycopg import (                     # noqa
ImportError: dlopen(/usr/local/anaconda3/envs/labelstudio/lib/python3.8/site-packages/psycopg2/_psycopg.cpython-38-darwin.so, 0x0002): Library not loaded: @rpath/libpq.5.dylib
  Referenced from: <21E21615-F706-3B33-B7B0-403832027FDF> /usr/local/anaconda3/envs/labelstudio/lib/python3.8/site-packages/psycopg2/_psycopg.cpython-38-darwin.so
  Reason: tried: '/usr/local/anaconda3/envs/labelstudio/lib/libpq.5.dylib' (no such file), '/System/Volumes/Preboot/Cryptexes/OS/usr/local/anaconda3/envs/labelstudio/lib/libpq.5.dylib' (no such file), '/usr/local/anaconda3/envs/labelstudio/lib/libpq.5.dylib' (no such file), '/System/Volumes/Preboot/Cryptexes/OS/usr/local/anaconda3/envs/labelstudio/lib/libpq.5.dylib' (no such file), '/usr/local/anaconda3/envs/labelstudio/lib/libpq.5.dylib' (no such file), '/System/Volumes/Preboot/Cryptexes/OS/usr/local/anaconda3/envs/labelstudio/lib/libpq.5.dylib' (no such file), '/usr/local/anaconda3/envs/labelstudio/lib/libpq.5.dylib' (no such file), '/System/Volumes/Preboot/Cryptexes/OS/usr/local/anaconda3/envs/labelstudio/lib/libpq.5.dylib' (no such file), '/usr/local/anaconda3/envs/labelstudio/bin/../lib/libpq.5.dylib' (no such file), '/usr/local/anaconda3/envs/labelstudio/bin/../lib/libpq.5.dylib' (no such file), '/System/Volumes/Preboot/Cryptexes/OS@rpath/libpq.5.dylib' (no such file), '/usr/local/anaconda3/envs/labelstudio/lib/libpq.5.dylib' (no such file), '/System/Volumes/Preboot/Cryptexes/OS/usr/local/anaconda3/envs/labelstudio/lib/libpq.5.dylib' (no such file), '/usr/local/anaconda3/envs/labelstudio/lib/libpq.5.dylib' (no such file), '/System/Volumes/Preboot/Cryptexes/OS/usr/local/anaconda3/envs/labelstudio/lib/libpq.5.dylib' (no such file), '/usr/local/anaconda3/envs/labelstudio/lib/libpq.5.dylib' (no such file), '/System/Volumes/Preboot/Cryptexes/OS/usr/local/anaconda3/envs/labelstudio/lib/libpq.5.dylib' (no such file), '/usr/local/anaconda3/envs/labelstudio/lib/libpq.5.dylib' (no such file), '/System/Volumes/Preboot/Cryptexes/OS/usr/local/anaconda3/envs/labelstudio/lib/libpq.5.dylib' (no such file), '/usr/local/anaconda3/envs/labelstudio/bin/../lib/libpq.5.dylib' (no such file), '/usr/local/anaconda3/envs/labelstudio/bin/../lib/libpq.5.dylib' (no such file), '/usr/local/lib/libpq.5.dylib' (no such file), '/usr/lib/libpq.5.dylib' (no such file, not in dyld cache)
https://o227124.ingest.sentry.io:443 "POST /api/5820521/store/ HTTP/1.1" 200 41```

</details>

**Comment by makseq** [comment]
You need to install the libpq library on your Mac to resolve this error. You can do this using the Homebrew package manager. If you don't have Homebrew installed, install it by running the following command:

```
/bin/bash -c "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/HEAD/install.sh)"
```

Next, install libpq:

```
brew install libpq
```

After installing libpq, make sure your system knows where to find the library. You can do this by adding the appropriate path to your DYLD_FALLBACK_LIBRARY_PATH environment variable. Add the following line to your ~/.zshrc or ~/.bashrc file (depending on which shell you use):

```
export DYLD_FALLBACK_LIBRARY_PATH="/usr/local/opt/libpq/lib:$DYLD_FALLBACK_LIBRARY_PATH"
```

After adding the line, restart your terminal or run source ~/.zshrc or source ~/.bashrc to apply the changes.

Now, you should be able to import psycopg2 without any issues. If you still face problems, consider reinstalling the psycopg2 package:

```
pip uninstall psycopg2
pip install psycopg2
```

**Comment by makseq** [comment]
This issue will be fixed in upcoming release 1.7.3.

**Comment by makseq** [comment]
Another thing you can try:

```
conda install psycopg2
```
